File: codeforces/99.floor_number_[1426A]/floor_number.py
# Author: José Rodolfo (jric2002)
t = int(input())
while (t):
  n, x = map(int, input().split(" "))
  f = None
  if (n <= 2):
    f = 1
  else:
    if ((n - 2) <= x):
      f = 2
    else:
      f = 1
      i = 3
      while (i <= n):
        f += 1
        i += x
  print(f)
  t -= 1

File: codeforces/99.floor_number_[1426A]/floor_number.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int t, n, x, f;
  cin >> t;
  while (t--) {
    cin >> n >> x;
    if (n <= 2) {
      f = 1;
    }
    else {
      if ((n - 2) <= x) {
        f = 2;
      }
      else {
        f = 1;
        for (unsigned short int i = 3; i <= n; i = i + x) {
          f++;
        }
      }
    }
    cout << f << endl;
  }
  return 0;
}